charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml

```diff
@@ -270,6 +270,9 @@ spec:
         - name: fedora-rhel6-alpine-openelec-cabundle
           mountPath: /etc/ssl
           readOnly: true
+        - name: usr-share-ca-certificates
```

Comment:
This changes makes g/g incompatible with all provider-aws versions other than provider-aws@master. 

```
status:
  lastErrors:
  - description: 'task "Deploying Kubernetes API server" failed: retry failed with
      context deadline exceeded, last error: failed to apply manifests: 1 error occurred:
      could not apply object of kind "Deployment" "shoot--i331370--aws-local/kube-apiserver":
      Deployment.apps "kube-apiserver" is invalid: [spec.template.spec.containers[0].volumeMounts[17].mountPath:
      Invalid value: "/etc/ssl": must be unique, spec.template.spec.containers[0].volumeMounts[18].mountPath:
      Invalid value: "/usr/share/ca-certificates": must be unique]'
```

There should be at least a release note or warning about that...